benchmarks/asv/app_suite.py

```diff
@@ -15,5 +15,17 @@
 Define asv benchmark suite that estimates the speed of applications.
 """
 
+from ..benchmark_functions.vqe import benchmark_vqe
 
 
+class VQEEvaluation:
```

Comment:
```suggestion
class VQE:
```
I think this reads better and is still clear